addons/test_mail/models/test_mail_models.py

```diff
@@ -26,6 +26,7 @@ class MailTestStandard(models.Model):
     email_from = fields.Char()
     user_id = fields.Many2one('res.users', 'Responsible', track_visibility='onchange')
     umbrella_id = fields.Many2one('mail.test', track_visibility='onchange')
+    company_id = fields.Many2one('res.company')
```

Comment:
Are you sure there is no problem with that?

Reply:
What could go wrong when adding a field in stable? ;)
It is tolerated in test-exclusive modules
I speak under the control of @KangOl

Reply:
It's kinda tolerated for `test_*` modules if there is not other way...

Reply:
Hacking a very nice test model in stable makes me very very sad.

I understand you don't really like me ...